src/abstractdataframe/io.jl

```diff
@@ -127,8 +127,8 @@ julia> show(stdout, MIME("text/csv"), DataFrame(A=1:3, B=["x", "y", "z"]))
 """
 Base.show(io::IO, mime::MIME, df::AbstractDataFrame)
 Base.show(io::IO, mime::MIME"text/html", df::AbstractDataFrame;
-          summary::Bool=true, eltypes::Bool=true) =
-    _show(io, mime, df, summary=summary, eltypes=eltypes)
+          summary::Bool=true, eltypes::Bool=true, kwargs...) =
```

Comment:
Should we check if `kwargs` contains `rowid` or `title`, and if it does throw an error?

Reply:
I do not see a reason here to check. What is the problem of this version?

Reply:
User could pass `rowid="some crazy string"` and break the logic of `rowid` handling (this kwarg is not exposed to the user as it is used to determine if `DataFrameRow` was asked to be printed).

Similarly when user passes `title="This is not a data frame"` then instead of the expected auto-generated title a custom title is printed. Again - we did not expose this option to the user - do you think we should?

Reply:
Hum, the rowid can be a problem. However, title is exactly the keyword used by PrettyTables to print the title. If we do this, the user will not be able to customize the title as they can with the other parameters. What do you want to do here?

Reply:
> However, title is exactly the keyword used by PrettyTables to print the title.

This is what I initially thought, but later I saw in https://github.com/JuliaData/DataFrames.jl/pull/3096/files#diff-816ec4315049c46513c56fa4009c044833c70607075e8156e999811995687005R270:
```
top_left_str = title
```
so I thought that `top_left_str` is the kwarg used to set title.

This is different from what you do in text/plain in https://github.com/JuliaData/DataFrames.jl/blob/372c9fb890c9e4d30699bc3ad56daf7482115e96/src/abstractdataframe/show.jl#L282

That is why I thought that `title` has a special treatment in html output. So what is the situation here?

Reply:
Oops, you are 100% right. I completely forgot I did this modification so that we can have the omitted cell summary in the top right. Furthermore, in the future, I will use title to be the table caption. So you are right. I will check the presence of those arguments and throw errors.

Reply:
Thank you!